account_cash_discount_write_off/README.rst

```diff
@@ -0,0 +1,87 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Account Cash Discount Write Off
+===============================
+
+Create an automatic write-off for payment with discount on the payment order
+confirmation. If the cash discount amount is computed based on the total
+amount, the created write-off will also contains tax adjustments. This
```

Comment:
Better have an independent flag to control tax adjustments.

Reply:
But would it be coherent to compute the discount based on the price (excluding taxes) and having the "Tax Adjustments" checked. What should we do in theses situations ?